includes/abstracts/abstract-wc-shipping-method.php

```diff
@@ -10,78 +10,218 @@
  * Extended by shipping methods to handle shipping calculations etc.
  *
  * @class       WC_Shipping_Method
- * @version     2.3.0
+ * @version     2.6.0
  * @package     WooCommerce/Abstracts
  * @category    Abstract Class
  * @author      WooThemes
  */
 abstract class WC_Shipping_Method extends WC_Settings_API {
 
-	/** @var string Unique ID for the shipping method - must be set. */
-	public $id;
+	/**
+	 * Features this method supports. Possible features used by core:
+	 * - shipping-zones Shipping zone functionality + instances
+	 * - instance-settings Instance settings screens.
+	 * - settings Non-instance settings screens. Enabled by default for BW compatibility with methods before instances existed.
+	 * @var array
+	 */
+	public $supports             = array( 'settings' );
```

Comment:
I'm not a fan of these being aligned since they are not on lines directly above/below each other (comments between) - so it looks a bit weird.